src/DynamoCore/Models/RecordableCommands.cs

```diff
@@ -970,6 +984,7 @@ where xmlNode.Name.Equals("ModelGuid")
             internal IEnumerable<Guid> ModelGuids { get { return modelGuids; } }
             internal string Name { get; private set; }
             internal string Value { get; private set; }
+            internal WorkspaceModel TargetWorkspace { get; private set; } 
```

Comment:
For the same reasoning as above, this should be `Guid WorkspaceGuid`. If you noticed, the attribute names used in most other recordable commands are the same as the properties themselves, so when we serialize this command, it would end up as `WorkspaceGuid`. This is just to align better with other commands.